src/lib/merkle_ledger_tests/test_database.ml

```diff
@@ -163,6 +181,78 @@ let%test_module "test functor on in memory databases" =
                | `Added -> ()
                | `Existed -> MT.set mdb location account )
 
+      let%test_unit "If the entire database is full, let \
+                     addresses_and_accounts = \
+                     get_all_accounts_rooted_at_exn(address) in \
+                     set_batch_accounts(addresses_and_accounts) won't cause \
+                     any changes" =
```

Comment:
This looks like a duplicate from `Database`. Can you add this function in the util module to make this reusable?